.github/BECOMING_A_CORE_CONTRIBUTOR.md

```diff
@@ -17,53 +18,45 @@ With that said, the Lightning team will be diverse and a reflection of an inclus
-  Often, these issues require some sort of bug fix, document clarification or new functionality to be scoped out.
+- Help users so they have an amazing experience. Issues range from questions from new people getting into DL to questions from researchers about doing something esoteric with Lightning.
+  They often require some sort of bug fix, document clarification, or new functionality to be scoped out. You can help them solve their issues and guide them to completion.
 
-- To become a core member you must resolve at least 10 Github issues which align with the API design goals for Lightning. By the end of these 10 issues I should feel comfortable in the way you answer user questions
-  Pleasant/helpful tone.
+- To weigh in on discussions in a timely fashion. Most importantly, on the RFCs (request for comments) that will shape the future of Lightning.
+  There are some big decisions which the project must make. For these, we expect core contributors to have something meaningful to add, especially if it’s their area of expertise.
 
-- Can abstract from that issue or bug into functionality that might solve other related issues or makes the platform more flexible.
+- Propose your own RFCs that align with the API design goals for Lightning.
+
+- Can abstract from an issue or bug into functionality that might solve other related issues or make the framework more flexible.
 
 - Don’t make users feel like they don’t know what they’re doing. We’re here to help and to make everyone’s experience delightful.
 
+- Help out with critical bugs. Nobody likes bugs so you'll be a hero if you fix them!
+
 ### Pull requests
 
-- Here we need to ensure the code that enters Lightning is high quality. For each PR we need to:
-- Make sure code coverage does not decrease
-- Documents are updated
-- Code is elegant and simple
-- Code is NOT overly engineered or hard to read
-- Ask yourself, could a non-engineer understand what’s happening here?
-- Make sure new tests are written
-- Is this NECESSARY for Lightning? There are some PRs which are just purely about adding engineering complexity which have no place in Lightning.
-  Guidance
-- Some other PRs are for people who are wanting to get involved and add something unnecessary. We do want their help though! So don’t approve the PR, but direct them to a Github issue that they might be interested in helping with instead!
-- To be considered for core contributor, please review 10 PRs and help the authors land it on master. Once you've finished the review, ping me
-  for a sanity check. At the end of 10 PRs if your PR reviews are inline with expectations described above, then you can merge PRs on your own going forward,
-  otherwise we'll do a few more until we're both comfortable :)
+- Pull requests are the evolutionary mechanism of Lightning, so quality is extremely important. Make sure contributors adhere to the guidelines described in the [contributing section](CONTRIBUTING.md#Pull-request).
```

Comment:
```suggestion
- Pull requests are the evolutionary mechanism of Lightning, so quality is extremely important. Make sure contributors adhere to the guidelines described in the [contributing section](CONTRIBUTING.md#Pull-request).
Even you are not yet core, you can still read, comment, and mark your reviewer's decision. It will much boost our review process and also show us your good work.
```